microsoft-365/compliance/dlp-microsoft-teams.md

```diff
@@ -26,7 +26,10 @@ If your organization has Microsoft Purview Data Loss Prevention (DLP), you can d
     > [!NOTE]
     > DLP for Microsoft Teams blocks sensitive content when shared with Microsoft Teams users who have:<br/>- [guest access](/MicrosoftTeams/guest-access) in teams and channels; or<br/>- [external access](/MicrosoftTeams/manage-external-access) in meetings and chat sessions. <p>DLP for external chat sessions will only work if both the sender and the receiver are in Teams Only mode and using [Microsoft Teams native federation](/microsoftteams/manage-external-access). DLP for Teams does not block messages in [interop](/microsoftteams/teams-and-skypeforbusiness-coexistence-and-interoperability#interoperability-of-teams-and-skype-for-business) with Skype for Business or non-native federated chat sessions.
 
-- **Example 2: Protecting sensitive information in documents**. Suppose that someone attempts to share a document with guests in a Microsoft Teams channel or chat, and the document contains sensitive information. If you have a DLP policy defined to prevent this, the document won't open for those users. Your DLP policy must include SharePoint and OneDrive in order for protection to be in place. This is an example of DLP for SharePoint that shows up in Microsoft Teams, and therefore requires that users are licensed for Office 365 DLP (included in Office 365 E3), but does not require users to be licensed for Office 365 Advanced Compliance.)
+- **Example 2: Protecting sensitive information in documents**. Suppose that someone attempts to share a document with guests in a Microsoft Teams channel or chat, and the document contains sensitive information. If you have a DLP policy defined to prevent this, the document won't open for those users. Your DLP policy must include SharePoint and OneDrive in order for protection to be in place. This is an example of DLP for SharePoint that shows up in Microsoft Teams, and therefore requires that users are licensed for Office 365 DLP (included in Office 365 E3), but does not require users to be licensed for Office 365 Advanced Compliance. 
+   
+    In case you don’t want to have a separate DLP policy for SharePoint and OneDrive or want the files shared in Teams to be evaluated under same conditions, you can extend the Teams DLP policy to cover these locations (SharePoint and OneDrive) by selecting ‘Automatic file protection’ from the banner in DLP -> Policies. This will enable all the files shared in Teams chats and channels to be protected with the same rules* that apply to Teams messages. Once enabled, the extension will apply to all existing as well as future Teams DLP policies, thereby not making it mandatory to create separate policies for SharePoint and OneDrive for file protection. 
```

Comment:
```suggestion
    In case you don’t want to have a separate DLP policy for SharePoint and OneDrive or want the files shared in Teams to be evaluated under the same conditions, you can extend the Teams DLP policy to cover these locations (SharePoint and OneDrive) by selecting ‘Automatic file protection’ from the banner in DLP -> Policies. This will enable all the files shared in Teams chats and channels to be protected with the same rules* that apply to Teams messages. Once enabled, the extension will apply to all existing as well as future Teams DLP policies, thereby not making it mandatory to create separate policies for SharePoint and OneDrive for file protection. 
```

Reply:
@tanaybaldua 
MS style guide doesn't allow '*' and footnotes.  There are also other style and clarity issues.

@JohanFreelancer9  can you please address the use of ' , ->, * Delete the text on line 31 from "In case...through...same conditions,"  This section should start with "You can extend"  The text in the existing footnote needs to be incorporated into the text or separated out as  !NOTE element.  If you'd rather I do these tasks, that's ok, let me know.

Reply:
@chrfox, thank you for your comments and guidance. Much appreciated. I have incorporated your comments in my suggested changes. Please let me know if anything else is needed.